### DISCUSSION TITLE
🤷‍♂️哔哩的商店下载渠道将废弃

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1716] [通知]
最近 Microsoft Store 的审核明显严了很多。

哔哩作为 BiliBili 的第三方应用，在应用名称、logo、功能体验上和官方应用极为相似，因此遭到了应用商店的屡次拒绝。

对于这一点，我表示理解。

但同时也意味着以下事情：

1. 用户只能从 Github 获取最新的应用包
2. Xbox 用户已经在事实上无法获取到哔哩的更新了

我目前的业余精力并不在哔哩项目上了，希望大家理解一二

**Comment by neigechengzi** [comment]
真可惜

**Comment by FurAoser** [comment]
xbox平台的优化一直很差，放弃xbox不一定是个不好的选择
